lib/src/layer/polygon_layer/painter.dart

```diff
@@ -91,11 +122,27 @@ class _PolygonPainter extends CustomPainter {
       // ignore: deprecated_member_use_from_same_package
       if (polygon.isFilled ?? true) {
         if (polygon.color != null) {
```

Comment:
Not a change of this pull request but am I correct that these two if statements can be merged together?

Reply:
Yes, they can be. I left it like this to make it clearer/easier to remove once the deprecation is removed. I can change it (and elsewhere) if you prefer that.